package.json

```diff
@@ -8,9 +8,9 @@
     "url": "https://github.com/hawkular/hawkinit"
   },
   "scripts": {
-    "test": "echo \"WARN: no test specified; todo: add some tests :]\" && exit 0",
+    "test": "mocha specs/*",
```

Comment:
You are correct, see here https://travis-ci.org/hawkular/hawkinit/jobs/237385328#L575-L632